extensions/network/net-systemd-timesyncd.sh

```diff
@@ -0,0 +1,14 @@
+#
+# Extension to manage network time synchronization with systemd-timesyncd
+#
+function add_host_dependencies__install_systemd-timesyncd() {
```

Comment:
`add_host_dependencies` is not adequate for this. use `extension_prepare_config` or such.